snippets/logging-http-sys-input-support.adoc

```diff
@@ -0,0 +1,19 @@
+// Text snippet included in the following assemblies:
+//
+// Text snippet included in the following modules:
+// * log-collector-rsyslog-server
+// * 
+
+:_mod-docs-content-type: SNIPPET
+
+[IMPORTANT]
+====
+{feature-name} is only supported for the following scenarios:
+
+* Logging is installed on {hcp-capital}. 
```

Comment:
```suggestion
* Logging is installed on {hcp}. 
```

hosted control planes should start with lowercase when not placed at the start of the heading/sentence/title.
See the term glossary for usage: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#hosted-control-planes